packages/test_packages_common.py

```diff
@@ -26,17 +27,10 @@ def built_packages() -> List[str]:
 
     This functions lists the names of the .data files in the build/ directory.
     """
-    if not BUILD_DIR.exists():
+    packages_json_path = BUILD_DIR / "packages.json"
+    if not packages_json_path.exists():
         return []
-    registered_packages_ = registered_packages()
-    packages = []
-    for fpath in os.listdir(BUILD_DIR):
-        if not fpath.endswith(".data"):
-            continue
-        name = fpath.split(".")[0]
-        if name in registered_packages_:
-            packages.append(name)
-    return packages
+    return list(json.loads(packages_json_path.read_text())["packages"].keys())
```

Comment:
Would it be hard to avoid having two same functions `built_packages` in both here and in `conftest.py`?
I'm not sure which would be the best place to put this function (maybe somewhere inside `pyodide-build`?)

Reply:
We can just import it directly from conftest actually.